src/OmniSharp.Roslyn.CSharp/Services/Buffer/ChangeBufferService.cs

```diff
@@ -0,0 +1,26 @@
+using System.Composition;
+﻿using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using OmniSharp.Mef;
+using OmniSharp.Models;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Buffer
+{
+    [OmniSharpHandler(typeof(RequestHandler<ChangeBufferRequest, object>), LanguageNames.CSharp)]
```

Comment:
This would look worse in F# where you don't expect to see so much type cluttering.